### DISCUSSION TITLE
anova.gllvm nested models and coefplot species coefficients

### DISCUSSION THREAD

**Discussion by mike-kratz** [JenniNiku/gllvm#114] [Q&A] [answered]
Hi there!

In addition to a Procrustes correlation + test between my unconstrained and concurrent ordinations, I would also like to implement a likelihood-ratio test to compare the two (the more evidence the merrier!). Would an unconstrained ordination be considered nested within a concurrent ordination if all setting remained the same (i.e. same `row.effect`, `family`, `offset`, etc.)? I know this is necessary for the resulting p-value to have any meaning. 

Also as a separate question, is there any way to extract the taxa coefficient for environmental variables that are used in the `coefplot` function? I am working with microbial data and it is not possibly to manually view which taxa had the strongest and weakest responses from the plot since there are too many taxa on the y-axis. 

Thank you,

Mike Kratz 
![Coefficient Plot](https://user-images.githubusercontent.com/121577345/217154835-659579a4-94c1-440c-b6fb-84634b5e9627.png)

**Comment by BertvanderVeen** [comment, marked as answer]
Thanks for another excellent question, Mike. Unconstrained and concurrent ordination are nested, constrained and concurrent ordination are nested, unconstrained and constrained ordination are not nested. So it should be fine to compare your unconstrained and concurrent ordinations with a likelihood ratio test!

You can extract the taxa coefficients as: `model$params$LvXcoef%*%t(model$params$theta)` and their standard errors with `t(gllvm:::RRse(model))`.

**Comment by mike-kratz** [comment]
@BertvanderVeen Great, thank you for your quick response and code! I appreciate all the help you've provided me and hopefully others will benefit from seeing these posts!

Have a great rest of your week,

Mike Kratz

**Reply by gerverska** [reply]
@mike-kratz Thanks for reaching out about this--out of curiosity, how many samples do you have?

**Reply by mike-kratz** [reply]
@gerverska Sorry for the delayed reply. I had 28 samples
